library/src/main/java/com/sothree/slidinguppanel/positionhelper/ScrollPositionHelper.kt

```diff
@@ -0,0 +1,16 @@
+package com.sothree.slidinguppanel.positionhelper
+
+import android.view.View
+
+interface ScrollPositionHelper{
```

Comment:
It's not important, but I guess this file needs a format. `...lper {`

Reply:
> What do you mean ?

may delete this paragraph in `setting.gradle` when deploying to maven or others
```gradle
include ':demo'
```

Reply:
I already did the format. My indicator was the missing blank between `ScrollPositionHelper` and `{`

Reply:
> may delete this paragraph in `setting.gradle` when deploying to maven or others
> 
> ```groovy-gradle
> include ':demo'
> ```

No action required, only `library` will be deployed  https://github.com/hannesa2/AndroidSlidingUpPanel/blob/master/library/build.gradle#L4

Reply:
> https://github.com/hannesa2/AndroidSlidingUpPanel/blob/master/library/build.gradle#L4

sry, this is my problem